pages/upgrade/graylog-3.1.rst

```diff
@@ -4,3 +4,125 @@ Upgrading to Graylog 3.1.x
 
 .. _upgrade-from-30-to-31:
 
+.. contents:: Overview
+   :depth: 3
+   :backlinks: top
+
+Views & Extended Search
+=======================
+
+The Views and Extended Search feature has been open-sourced in this vesion. (except the support for parameters) It was only accessible in Graylog Enterprise in Graylog 3.0.
+
+
+HTTP API Changes
+----------------
+
+The following Views related HTTP API paths changed due to the migration to open source:
+
++--------------------------------------------------------+-----------------------+
+| Old Path                                               | New Path              |
++========================================================+=======================+
+| ``/api/plugins/org.graylog.plugins.enterprise/views``  | ``/api/views``        |
++--------------------------------------------------------+-----------------------+
+| ``/api/plugins/org.graylog.plugins.enterprise/search`` | ``/api/views/search`` |
++--------------------------------------------------------+-----------------------+
+
+Configuration File Changes
+--------------------------
+
+The following views related configuration file settings changed due to the migration to open source:
+
++------------------------------------------+------------------------------+
+| Removed Setting                          | New Setting                  |
++==========================================+==============================+
+| ``enterprise_search_maximum_search_age`` | ``views_maximum_search_age`` |
++------------------------------------------+------------------------------+
+
+
+Alerts
+======
+
+The old Alerts system has been replaced by an Alerts & Events system in Graylog 3.1.
+
+Existing alerts will be automatically migrated to the new system when the Graylog 3.1 server starts for the first time.
+The migration will log the number of migrated legacy alert conditions and alarm callbacks::
+
+    2019-08-05T10:36:06.404Z INFO  [V20190722150700_LegacyAlertConditionMigration] Migrated <2> legacy alert conditions and <2> legacy alarm callbacks
+
+Alarm Callback Plugins
+----------------------
+
+The new Alerts & Events system is supporting the execution of legacy Alarm Callback plugins for now. We recommend to switch event definitions over to event notifications, though. At some point in the future support for legacy Alarm Callback plugins will be removed.
```

Comment:
Please note, that the data sent via a legacy Alarm Callback might be slightly different than via the old Alerts system.
If you've built automation on top of alerts, you might want to check that everything still works after the migration.